In [1]:
import pandas as pd
import seaborn as sns

In [2]:
train = pd.read_csv("train2.csv") # loading the train1 dataset from step_2


In [3]:
train.head()

order_no       user_id vehicle_type  platform_type customer_type  \
0   Order_No_4211   User_Id_633         Bike              3      Business   
1  Order_No_25375  User_Id_2285         Bike              3      Personal   
2   Order_No_1899   User_Id_265         Bike              3      Business   
3   Order_No_9336  User_Id_1402         Bike              3      Business   
4  Order_No_27883  User_Id_1737         Bike              1      Personal   

   placed_day  placed_wkday          placed_time  confirmed_day  \
0           9             5  1900-01-01 09:35:46              9   
1          12             5  1900-01-01 11:16:16             12   
2          30             2  1900-01-01 12:39:25             30   
3          15             5  1900-01-01 09:25:34             15   
4          13             1  1900-01-01 09:55:18             13   

   confirmed_wkday  ... pickup_lat  pickup_long  delivered_lat delivered_long  \
0                5  ...  -1.317755    36.830370      -1.300406      36.829741   
1                5  ...  -1.351453    36.899315      -1.295004      36.814358   
2                2  ...  -1.308284    36.843419      -1.300921      36.828195   
3                5  ...  -1.281301    36.832396      -1.257147      36.795063   
4                1  ...  -1.266597    36.792118      -1.295041      36.809817   

       Rider Id  time_pickup_to_arrival No_Of_Orders   Age  Average_Rating  \
0  Rider_Id_432                     745         1637  1309            13.8   
1  Rider_Id_856                    1993          396   339            13.6   
2  Rider_Id_155                     455         1023   242            12.5   
3  Rider_Id_855                    1341          886   283            14.5   
4  Rider_Id_770                    1214         2311   872            14.1   

  No_of_Ratings  
0           549  
1            69  
2           114  
3           113  
4           533  

[5 rows x 32 columns]

In [11]:
train.columns

Index(['order_no', 'user_id', 'vehicle_type', 'platform_type', 'customer_type',
       'placed_day', 'placed_wkday', 'placed_time', 'confirmed_day',
       'confirmed_wkday', 'confirmed_time', 'arrive_pickup_day',
       'arrive_pickup_wkday', 'arrive_pickup_time', 'pickup_day',
       'pickup_wkday', 'pickup_time', 'delivered_day', 'delivered_wkday',
       'delivered_time', 'distance_km', 'temp', 'pickup_lat', 'pickup_long',
       'delivered_lat', 'delivered_long', 'Rider Id', 'time_pickup_to_arrival',
       'No_Of_Orders', 'Age', 'Average_Rating', 'No_of_Ratings'],
      dtype='object')

## Categorical Variables

In [8]:
train['confirmed_wkday'].value_counts() #seeing how many orders are placed within each weekday 

4    4229
5    3993
2    3959
3    3823
1    3788
6    1223
7     186
Name: confirmed_wkday, dtype: int64

From here there are:
<br>
3788 orders on Monday
<br>
3959 orders on Tuesday
<br>
3823 orders on Wednesday 
<br>
4229 orders on Thursday
<br>
3993 orders on Friday
<br>
1223 orders on Saturday
<br>
186 orders on Sunday